packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.tsx

```diff
@@ -186,20 +202,11 @@ const AccountRecoveryConfirmKey = (_: RouteComponentProps) => {
       checkRecoveryKey(submitData);
     }
   };
-
-  if (linkStatus === LinkStatus.damaged || requiredParams === null) {
-    return <ResetPasswordLinkDamaged />;
-  }
-
-  if (linkStatus === LinkStatus.expired) {
+  if (showLoadingSpinner) {
     return (
-      <LinkExpiredResetPassword
-        email={requiredParams.email}
-        {...{ viewName }}
-      />
+      <LoadingSpinner className="bg-grey-20 flex items-center flex-col justify-center h-screen select-none" />
```

Comment:
Not in this PR, but feels like we should revisit our default class names for the spinner at some point or have an option for this state since we use it frequently 🤔